src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/person/PersonKeyService.kt

```diff
@@ -39,7 +33,7 @@ class PersonKeyService(
       personEntity,
       mapOf(EventKeys.UUID to personKey.personId.toString()),
     )
-    return personKeyRepository.saveAndFlush(personKey)
+    return personKeyRepository.save(personKey)
```

Comment:
it is a suprising side effect of a method called `getXX` that it should persist. I see that it already did this